lib/app/common/extensions/enums/enums_formatter_ext.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:score_live/app/common/extensions/enums.dart';
+import 'package:score_live/presentation/constants/app_const_variables.dart';
+
+extension EnumsFormatter on StatsSwitch {
+  String statsSwitchHeaderFormatter() {
```

Comment:
Please convert this to a getter and use switch statement - looks cleaner

Reply:
good point